hoomd/mpcd/CellList.cc

```diff
@@ -138,23 +144,16 @@ void mpcd::CellList::compute(uint64_t timestep)
             m_embed_cell_ids.resize(m_embed_group->getNumMembers());
             }
 
-        bool overflowed = false;
-        do
-            {
-            buildCellList();
-
-            overflowed = checkConditions();
-
-            if (overflowed)
-                {
-                reallocate();
-                resetConditions();
-                }
-            } while (overflowed);
+        // bin particles and compute cell properties
+        buildCellList();
+        checkConditions();
+        m_property_sum = true;
```

Comment:
What is the purpose of this variable? I see it floating around it a few places, but my feeling is that it isn't needed.